pkg/controller/controlplane/valuesprovider.go

```diff
@@ -417,9 +417,24 @@ func getConfigChartValues(
 		"subNetworkName": subNetworkName,
 		"zone":           cpConfig.Zone,
 		"nodeTags":       cp.Namespace,
+		"alphaFeatures":  getAlphaFeatuesValues(cpConfig.CloudControllerManager),
 	}, nil
 }
 
+func getAlphaFeatuesValues(ccmConfig *apisgcp.CloudControllerManagerConfig) []string {
+	var alphaFeatues = []string{}
+
+	if ccmConfig == nil {
+		return alphaFeatues
+	}
```

Comment:
```suggestion
	if ccmConfig == nil {
		return nil
	}
```

and move to the top?